CHANGELOG.rst

```diff
@@ -18,6 +18,7 @@ Release 4.1.0
 
 **Bugs**:
 
+* Added server namespace 1 #92, by @huebl
```

Comment:
It'd be better to fix the documentation in this PR as well. Give me a while and I'll do it.